west.yml

```diff
@@ -136,6 +138,10 @@ manifest:
     - name: trusted-firmware-m
       path: modules/tee/tfm
       revision: 58c70aeb9937c3b6b97a07cc654e48b2c56c258a
+    - name: sof
+      remote: thesofproject
```

Comment:
We do not accept pulling from other GitHub organizations. @nashif can create a module repository for you inside the zephyrproject-rtos org that can mirror the repo.

Reply:
Ah, OK.  This is exactly the kind of collision I was hoping to tease out.  But I'm not sure a Zephyr mirror really addresses the problem here.

SOF is their own project.  They want to do development in their own tree.  They want to use Zephyr (and **we** want them to use Zephyr!).  Having their code be mirrored into the Zephyr project is either going to force them to work in a foreign project's code environment, or it's going to force them to fork Zephyr for their own use and force us (heh, me) into an endless series of giant rebasing patches like this one.

To be clear: the reason this PR exists is that SOF already did fork Zephyr, the development there got quite diverged, and the changes (having been commited directly by a foreign project with different standards than we have) aren't going to be directly submittable to Zephyr without cleanup/squashing work like this.

Clearly the Right Thing here is to have SOF keep their own code in their own repo, and work on upstream Zephyr directly and not a fork.  How do we make that happen?

Should make the SOF application and west repo link be part of an out-of-tree application?  Does that even work with west?